flexmark-ext-tables/pom.xml

```diff
@@ -25,15 +25,10 @@
             <artifactId>flexmark-test-util</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>com.acrolinx.flexmark</groupId>
-            <artifactId>flexmark-ext-typographic</artifactId>
-            <scope>test</scope>
-        </dependency>
```

Comment:
I think we use the tables extension. The removal of this typographic extension will not affect how the tables extension works?

Reply:
It is/was a test dependency only, so the removal should not in any way affect the behavior of the main code.